src/rod_mechanics/external_forces/damping_force.cpp

```diff
@@ -4,41 +4,38 @@
 #include "rod_mechanics/soft_robots.h"
 #include "time_steppers/base_time_stepper.h"
 
-DampingForce::DampingForce(const std::shared_ptr<SoftRobots>& m_soft_robots, double m_viscosity)
-    : BaseForce(m_soft_robots), viscosity(m_viscosity) {
+DampingForce::DampingForce(const std::shared_ptr<SoftRobots>& soft_robots, double viscosity)
+    : BaseForce(soft_robots), viscosity(viscosity) {
 }
 
-DampingForce::~DampingForce() {
-}
+DampingForce::~DampingForce() = default;
```

Comment:
Just wondering. Why do we need to add = default here?

Reply:
It's essentially equivalent. It's suggested by clang-tidy to use `default` to explicitly show that there is no deconstruction logic.